workers.go

```diff
@@ -277,7 +277,7 @@ func (api *API) UploadWorker(ctx context.Context, rc *ResourceContainer, params
 
 	uri := fmt.Sprintf("/accounts/%s/workers/scripts/%s", rc.Identifier, params.ScriptName)
 	if params.DispatchNamespaceName != nil {
-		uri = fmt.Sprintf("/accounts/%s/workers/namespaces/%s/scripts/%s", rc.Identifier, *params.DispatchNamespaceName, params.ScriptName)
+		uri = fmt.Sprintf("/accounts/%s/workers/dispatch/namespaces/%s/scripts/%s", rc.Identifier, *params.DispatchNamespaceName, params.ScriptName)
```

Comment:
the documented public path should be `https://api.cloudflare.com/client/v4
/accounts/{account_identifier}/workers/dispatch_namespaces/{dispatch_namespace}/scripts/{script_name}` (docs: https://developers.cloudflare.com/api/operations/namespace-worker-script-upload-worker-module)

Reply:
updated in 504d02a83cb56ca799f1ad098c6eea43d3831d74

Reply:
I think that may have been an error; we tried it with `workers/dispatch/namespaces` via Postman and we got back a successful message. Or is the other documentation wrong?